app/controllers/MeetupController.js

```diff
@@ -109,6 +109,41 @@ class MeetupController {
     }
   }
 
+  /**
+   * get all users scheduled upcoming meetup
+   *
+   * @param {*} req
+   * @param {*} res
+   */
+  static async getAllRsvps(req, res) {
+    try {
+      // Get a single meet up record
+      const queryString = 'SELECT meetups.topic, meetups.location, meetups.happeningon, meetups.tags FROM meetups LEFT JOIN rsvps ON meetups.id = rsvps.meetupid WHERE rsvps.userid = $1 AND rsvps.response = $2';
```

Comment:
@abidex4yemi what happens if the meetup date has passed?

Reply:
Fixed